packages/template-retail-react-app/app/pages/product-detail/index.jsx

```diff
@@ -129,7 +129,7 @@ const ProductDetail = () => {
                 body: {
                     // NOTE: APi does not respect quantity, it always adds 1
                     quantity,
-                    productId: urlParams.get('pid') || productId,
+                    productId: variant?.productId || product?.id,
```

Comment:
This is a bug fix for making sure that adding to wishlist works correctly for a product set: you can add the _parent_ item to the wishlist or the individual _child_ items.